tests/models/test_default_evaluator.py

```diff
@@ -804,17 +809,24 @@ def example_custom_metric(_, __, tmp_path):
     )
 
     with TemporaryDirectory() as tmp_dir:
-        for img_ext in ("png", "jpg", "jpeg"):
+        for img_ext in img_formats:
             assert f"test_{img_ext}_artifact" in result.artifacts
             assert f"test_{img_ext}_artifact_on_data_breast_cancer_dataset.{img_ext}" in artifacts
             assert isinstance(result.artifacts[f"test_{img_ext}_artifact"], ImageEvaluationArtifact)
-            expected_fig = plt.figure(figsize=(8, 5), dpi=100)
+
+            fig = plt.figure(figsize=(fig_x, fig_y), dpi=fig_dpi)
             plt.plot([1, 2, 3])
-            expected_fig.savefig(path_join(tmp_dir, f"test.{img_ext}"), format=img_ext)
+            fig.savefig(path_join(tmp_dir, f"test.{img_ext}"), format=img_ext)
             plt.clf()
-            assert result.artifacts[f"test_{img_ext}_artifact"].content == Image.open(
-                path_join(tmp_dir, f"test.{img_ext}")
-            )
+
+            saved_img = Image.open(path_join(tmp_dir, f"test.{img_ext}"))
+            result_img = result.artifacts[f"test_{img_ext}_artifact"].content
+
+            for img in (saved_img, result_img):
```

Comment:
PIL's Image reader equivalency was comparing object hashes for the previous assertion. Since we're modifying rc_params values for matplotlib that change the image serialization, this test fails. I've change the validation to ensure that the correct image type, (approximate) resolution, and rendered size.